app/static/js/controllers/securityGroupController.js

```diff
@@ -130,5 +133,15 @@ angular.module('uluwatuControllers').controller('securityGroupController', ['$sc
             $scope.securitygroupForm = {};
         }
 
+        $scope.unShowErrorMessageAlert = function() {
```

Comment:
I agree, but this way it's consistent. It's called 'unShowErrorMessageAlert' everywhere.